config/src/config/gas_estimation_config.rs

```diff
@@ -13,6 +13,9 @@ use serde::{Deserialize, Serialize};
 pub struct GasEstimationConfig {
     /// A gate for computing GasEstimation. If false, just returns the default.
     pub enabled: bool,
+    /// Provide static values to override, e.g., Some((100, 100, 150)). If set, just returns these
+    /// values instead of computing the actual GasEstimation.
+    pub static_override: Option<(u64, u64, u64)>,
```

Comment:
Instead of a tuple could you make this a struct? While it might be obvious in the code, in the configs this will appear to be just three numbers. Being able to set them as named fields would be better.